rush.json

```diff
@@ -436,10 +442,16 @@
       "cyclicDependencyProjects": ["@microsoft/node-library-build", "@microsoft/rush-stack-compiler-3.5"]
     },
     {
-      "packageName": "@microsoft/api-documenter",
-      "projectFolder": "apps/api-documenter",
+      "packageName": "@rushstack/heft",
+      "projectFolder": "apps/heft",
       "reviewCategory": "libraries",
-      "shouldPublish": true
+      "shouldPublish": true,
+      "cyclicDependencyProjects": ["@rushstack/heft"]
+    },
+    {
+      "packageName": "@rushstack/rundown",
+      "projectFolder": "apps/rundown",
+      "reviewCategory": "libraries"
```

Comment:
You want to publish this project?

Reply:
🤔 Yeah okay, the tool is probably reasonably complete enough to publish an initial version.